### PR TITLE
Main page styling changes

### PR DESCRIPTION
**Pull request by WeissSa**
## Pull Request Summary
Changed CSS for main page to better match figma

- [x] Front end change

### How to manually test this PR (QA instructions)
- Look at FE and make sure it looks nice

image

### CODE REVIEW

File: front-end/src/components/main-page/MainPage.jsx [hunk 1]
```diff
@@ -34,41 +34,50 @@ class MainPage extends Component {
...
-      <div className="MainPage bg-purple-100 absolute inset-0" data-testid="main-page">
-        <div className="flex gap-y-7 w-full flex-col mt-[25vh]">
+      <div className="bg-purple-100 absolute inset-0 flex flex-row" data-testid="main-page">
+        <div className="justify-center flex">
+          <TranscriptUploadModal
+            show={this.state.showTranscriptUploadModal}
+            toggleModal={this.toggleTranscriptUploadModal}
+          />
+        </div>
+        <div className="flex w-2/3 flex-col mt-[25vh] gap-y-12">
```
Comment (KayleeLChan) [inline front-end/src/components/main-page/MainPage.jsx:44]: I love how you're able to center everything, especially with how difficult Tailwind makes it, we stan mt-[25vh]

**Review by KayleeLChan — APPROVED** [review]
She looks spicy especially after that roasting we had in discord, main page is now the perfect temp. You are a pro chef handling all the heat 💃 💃 💃

File: front-end/src/components/main-page/MainPage.jsx (hunk 1 above)
Comment (WeissSa) [reply front-end/src/components/main-page/MainPage.jsx:44]: It's probably not actually centered, but it is close enough that our eyes can't tell 🙈

File: back-end/package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "test": "jest --coverage",
     "test:watch": "jest --watch",
-    "start": "node server.js"
+    "start": "node index.js",
+    "build": "rm -rf build"
```
Comment (WeissSa) [inline back-end/package.json:11]: this new build command does not do anything significant

**Review by helias22 — APPROVED** [review]

**Pull request merged**